src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -1241,7 +1241,9 @@
     <channel-type id="sound_tone_play">
         <item-type>Number</item-type>
         <label>Sound tone play</label>
-        <description>The Sound tone that will play. The allowed tone numbers depend on the device</description>
+        <description>The Sound tone that will play. The allowed tone numbers depend on the device. 
+            The value 0 will stop any tone beeing played. 
```

Comment:
It might be nice to add these two values as options (ie 0 and 255).

Reply:
I have updated the pull request with your suggested changes